ffs-shared/rule/src/commonTest/kotlin/rule/RuleEvalTest.kt

```diff
@@ -226,6 +226,19 @@ class RuleEvalTest {
         assertFailsWith<IllegalArgumentException> { eval("""contains("+01:00", "+01:00")""") }
     }
 
+    @Test
+    fun range() {
```

Comment:
Let's add a couple of tests for edge cases:
- Inverted ranges (e.g., `[10:0]`)
- At the precision limit of Int/Long.

Reply:
About inverted ranges, what's expected of `contains[10:0, 8]`?

I think it's better to invalidate inverted ranges input and throw `IllegalArgumentException`

What do you think?

Reply:
> `contains[10:0, 8]`

This syntax is incorrect, right?

> I think it's better to invalidate inverted ranges input and throw IllegalArgumentException, to keep the rule format consistent within the dashboard 😊

Throwing on inverted ranges sounds good. 👍

> Should we support negative value in the Range? e.g. [-1000, 140]. Right now I can't think of a common use case that will require a negative value range for the feature flag.

I can't either, but doesn't support come for free just by using signed types (the default)?